src/index.js

```diff
@@ -115,6 +115,8 @@ class TargetCoreAPI {
   * Creates a new AB activity with the specified contents and returns the created activity.
   *
   * @param body {Object} Activity JSON.
+  * @param options {Object} sdk options
```

Comment:
I always thought for the `@param` tag it was the type then name based on the docs, looks like it works in this order as well, is it correct? optional args are to be enclosed in square brackets, so it should be `[options]` and if the optional arg has a default param it should be assigned, like so `[options={}]`

Similar comment to the rest of the JSDocs.

Reply:
done